<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>795</width>
    <height>453</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(213, 213, 213);
border-radius:10px;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QFrame" name="frame_superior">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>801</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(158, 158, 158);
border-top-left-radius:10px;
border-top-right-radius:10px;
border-bottom-right-radius: 0px;
border-bottom-left-radius: 0px;</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QPushButton" name="bt_close_anillo">
        <property name="geometry">
         <rect>
          <x>750</x>
          <y>10</y>
          <width>36</width>
          <height>32</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>iconos/cerrar.svg</normaloff>iconos/cerrar.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
       <widget class="QPlainTextEdit" name="plainTextEdit_4">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>0</y>
          <width>471</width>
          <height>51</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Cantarell</family>
          <pointsize>16</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">margin-left:20px;
margin-top:8px;
color: #FFFFFF;</string>
        </property>
        <property name="plainText">
         <string>Anillo Leds</string>
        </property>
       </widget>
       <widget class="QPushButton" name="bt_minimize_anillo">
        <property name="geometry">
         <rect>
          <x>690</x>
          <y>10</y>
          <width>36</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>iconos/minimizar.svg</normaloff>iconos/minimizar.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </widget>
      <widget class="QFrame" name="frame_OnOff_anillo">
       <property name="geometry">
        <rect>
         <x>640</x>
         <y>70</y>
         <width>120</width>
         <height>78</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QRadioButton" name="anilloOnOff">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>101</width>
          <height>51</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>ArialBlack</family>
          <pointsize>12</pointsize>
          <weight>1</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QRadioButton
{
	color: #2a2a2a;
	font: 87 14 pt &quot;ArialBlack&quot;;
}

QRadioButton::indicator
{
	width: 50px;
	height: 50px;
	border-radius: 25px;
}

QRadioButton::indicator:checked {
   background-color: #00ff00;

}

QRadioButton::indicator:unchecked 
{
	background-color: #da0037;
	
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QLabel" name="etiquetaAnilloOnOff">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>30</y>
          <width>41</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">margin-top: 6px;
color: red;</string>
        </property>
        <property name="text">
         <string>OFF</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>10</y>
          <width>51</width>
          <height>22</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Anillo</string>
        </property>
       </widget>
      </widget>
      <widget class="QFrame" name="frame_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>120</width>
         <height>80</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>41</width>
          <height>22</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Pin:</string>
        </property>
       </widget>
       <widget class="QLabel" name="etiquetaAnilloPin">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>30</y>
          <width>61</width>
          <height>22</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </widget>
      <widget class="QFrame" name="frame_Bucle_anillo">
       <property name="geometry">
        <rect>
         <x>640</x>
         <y>170</y>
         <width>120</width>
         <height>78</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QRadioButton" name="anilloBucle">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>101</width>
          <height>51</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>ArialBlack</family>
          <pointsize>12</pointsize>
          <weight>1</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QRadioButton
{
	color: #2a2a2a;
	font: 87 14 pt &quot;ArialBlack&quot;;
}

QRadioButton::indicator
{
	width: 50px;
	height: 50px;
	border-radius: 25px;
}

QRadioButton::indicator:checked {
   background-color: #00ff00;

}

QRadioButton::indicator:unchecked 
{
	background-color: #da0037;
	
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QLabel" name="etiquetaAnilloBucle">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>30</y>
          <width>41</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">margin-top: 6px;
color: red;</string>
        </property>
        <property name="text">
         <string>OFF</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>10</y>
          <width>51</width>
          <height>22</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Bucle</string>
        </property>
       </widget>
      </widget>
      <widget class="QGraphicsView" name="graphicsView">
       <property name="geometry">
        <rect>
         <x>125</x>
         <y>52</y>
         <width>511</width>
         <height>401</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="frameShape">
        <enum>QFrame::Panel</enum>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
